subprojects/docs/src/docs/userguide/announce_plugin.adoc

```diff
@@ -22,73 +22,54 @@ The Gradle announce plugin allows you to send custom announcements during a buil
 * https://sites.google.com/site/snarlapp/home[Snarl] (Windows)
 * http://growl.info/[Growl] (macOS)
 
-
-
 [[sec:announce_usage]]
-=== Usage
+== Usage
 
 To use the announce plugin, apply it to your build script:
 
-++++
-<sample id="useAnnouncePlugin" dir="announce" title="Applying the announce plugin">
-            <sourcefile file="build.gradle" snippet="use-plugin"/>
-        </sample>
-++++
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/announce/build.gradle[tag=use-plugin]
+----
+
+[[sec:announce_configuration]]
+== Configuration
+
+NOTE: See also the link:{groovyDslPath}/org.gradle.api.plugins.announce.AnnouncePluginExtension.html[AnnouncePluginExtension] class in the API documentation.
 
-Next, configure your notification service(s) of choice (see table below for which configuration properties are available):
+Configure your notification service(s) of choice (see table below for which configuration properties are available):
 
-++++
-<sample id="useAnnouncePlugin" dir="announce" title="Configure the announce plugin">
-            <sourcefile file="build.gradle" snippet="announce-plugin-conf"/>
-        </sample>
-++++
+[source.multi-language-sample,groovy]
```

Comment:
No header here either, but the "Configuration" header was moved from the bottom up here - not sure if that's intentional.

Reply:
It's fine - just pointing it out as a difference.  If it's intentional, then I think we can leave it.